lldb/source/Plugins/Process/Linux/NativeRegisterContextLinux_arm64dbreg.h

```diff
@@ -0,0 +1,29 @@
+//===-- NativeRegisterContextLinux_arm64dbreg.h -----------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// These functions are split out to be reused in NativeRegisterContextLinux_arm,
+// for supporting debugging 32bit processes on arm64.
```

Comment:
Add something like "When debugging 32-bit process on Arm64, 64-bit ptrace interfaces must be used.". To me, this is the key detail.